platformshconfig/config.py

```diff
@@ -423,3 +494,53 @@ def isset(self, config_property):
         if is_build_var or is_runtime_var or is_unprefixed_var:
             return config_property is not None
         return False
+
+    @staticmethod
+    def pymongo_formatter(credentials):
```

Comment:
I am not a Pythonist so I don't know what the convention is here...  Do the formatters need to be methods of the object?  Can they just be free-floating functions in the same file?  Would that make them easier to reference?

PHP doesn't really do that, but for Node and Go that's what I did since those are more forgiving of free-floating functions.

Reply:
They can be removed as object methods, so I can change that.